usaspending_api/etl/management/sql/c_file_linkage/update_file_c_linkages_piid.sql

```diff
@@ -10,7 +10,7 @@ SET
 		WHERE
 			UPPER(aw.piid) = UPPER(faba.piid)
 			AND
-			UPPER(aw.parent_award_piid) IS NOT DISTINCT FROM UPPER(faba.parent_award_id)
+			UPPER(aw.parent_award_piid) = UPPER(faba.parent_award_id)
```

Comment:
Does the predicate ever need to be true when both `aw.parent_award_piid` and `faba.parent_award_id` are NULL? If so the original SQL handled that appropriately